### PR TITLE
Contributors fix

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: lib/metrics/github.rb [hunk 1]
```diff
@@ -2,9 +2,21 @@
 
 module Metrics
   class Github
+    NUMBER_OF_PAGES_FOR_TOTAL_COUNT = 10
```
Comment (segiddins) [inline lib/metrics/github.rb]: can you rename this to something more concise?

Comment (k0nserv) [inline lib/metrics/github.rb]: Absolutely. Any suggestions?

File: lib/metrics/github.rb [hunk 2]
```diff
@@ -89,6 +106,34 @@ def reset_not_found(pod)
 
     private
 
+    def total_count_of(collection, client)
+      case collection
+      when :contributors
+        result = client.repos.contributors(per_page: NUMBER_OF_PAGES_FOR_TOTAL_COUNT)
```
Comment (floere) [inline lib/metrics/github.rb]: This part confuses me – it's `per_page` and then you have pages. Pages per page? Should it not be something like `ITEMS_PER_PAGE`?

Comment (k0nserv) [inline lib/metrics/github.rb]: Yes you are right, like @segiddins said the name is not good. It is the number of items per page

**Comment by orta** [comment]
Looks good to this guy.

**Pull request merged**